qa/util/src/main/java/io/camunda/qa/util/cluster/TestStandaloneCamunda.java

```diff
@@ -116,15 +127,17 @@ public TestStandaloneCamunda start() {
 
     operateProperties.getElasticsearch().setUrl(esURL);
     operateProperties.getZeebeElasticsearch().setUrl(esURL);
-
+    tasklistProperties.getElasticsearch().setUrl(esURL);
+    tasklistProperties.getZeebeElasticsearch().setUrl(esURL);
     return super.start();
   }
 
   @Override
   public TestStandaloneCamunda stop() {
     // clean up ES/OS indices
     LOGGER.info("Stopping standalone camunda test...");
-    ((TestElasticsearchSchemaManager) bean(SchemaManager.class)).deleteSchema();
+    ((TestOperateElasticsearchSchemaManager) bean(SchemaManager.class)).deleteSchema();
+    ((TestTasklistElasticsearchSchemaManager) bean(SchemaManager.class)).deleteSchema();
```

Comment:
one last thing, I think you will get `ClassCastException` here
You need to use `bean(io.camunda.tasklist.schema.manager.SchemaManager.class)`

Reply:
True :) for some reason my IntelliJ has not threw an error over this. But was running from the cache, which is strange. 

After clean and install I could validate this, and refactor :) Well noted.